caluma/caluma_workflow/events.py

```diff
@@ -29,6 +29,7 @@
 created_case = Signal()
 suspended_case = Signal()
 resumed_case = Signal()
+reopened_case = Signal()
```

Comment:
Not needed, that's the "old" way of doing it. No need to introduce a deprecated signal :)